### PR TITLE
Migrate to Capital Framework 4

### PR DESCRIPTION
**Pull request by anselmbradford**
## Removes

- pagination, buttons, media queries less files and other styles coming in via CFv4.

## Changes

- Adds atomic prefixes to relevant classes. 
- Updates `capital-framework` to version `4.8.1`.
- Migrates codebase to current lists, jump links, buttons, labels, and meta-header.

## Testing

1. `./frontend.sh` and compare the local site pages to the live site and look for any changes in appearance. Let me know if you see anything off.

### CODE REVIEW

File: cfgov/unprocessed/css/molecules/notification.less [hunk 1]
```diff
@@ -90,6 +90,7 @@
 
         color: @gray-20;
         font-size: unit( 18px / @base-font-size-px, em );
+        line-height: 1;
```
Comment (anselmbradford) [inline cfgov/unprocessed/css/molecules/notification.less]: Needed because cf-icon doesn't set a `line-height: 1` anymore, I think.

File: package.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
     "babel-preset-stage-2": "6.24.1",
     "banner-footer-webpack-plugin": "git://github.com/dtothefp/banner-footer-webpack-plugin.git",
     "browser-sync": "2.18.7",
-    "capital-framework": "3.6.1",
+    "capital-framework": "4.8.1",
```
Comment (Scotchester) [inline package.json]: I'll give this a more thorough review tomorrow, but we want to switch to requiring individual CF components at this point, to make single component updates possible and hopefully prevent getting as far behind as we are now in the future.

**Comment by anselmbradford** [comment]
Please pull latest.

File: cfgov/unprocessed/css/enhancements/forms.less [hunk 3]
```diff
@@ -248,7 +248,7 @@ select[multiple] {
     padding: 5px;
 }
 
-.form-label-header {
+.a-label__heading {
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/enhancements/forms.less]: These updates are already in CF right? Can you remove them here?

File: cfgov/unprocessed/css/molecules/multi-select.less [hunk 4]
```diff
@@ -93,7 +88,12 @@
             margin-right: unit(15px / @base-font-size-px, em);
         }
 
+        li:last-child {
+          margin-bottom: unit(5px / @base-font-size-px, em);
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/molecules/multi-select.less]: Minor: Add spaces within parens

File: cfgov/unprocessed/css/molecules/social-media.less [hunk 5]
```diff
@@ -61,17 +61,17 @@
         }
     }
 
-    .list {
+    .m-list {
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/molecules/social-media.less]: Could we use [`m-list__horizontal'](https://cfpb.github.io/capital-framework/components/cf-typography/#horizontal-list-modifier) instead? Seems like this is duplicating those styles.

File: cfgov/unprocessed/css/organisms/email-popup.less [hunk 6]
```diff
@@ -66,7 +66,7 @@
 
     .respond-to-max( @bp-xs-max, {
       background: none;
-      .m-form-field-with-button .btn {
+      .m-form-field-with-button .a-btn {
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/organisms/email-popup.less]: [This is a class now](https://github.com/cfpb/capital-framework/pull/483)

File: cfgov/unprocessed/css/organisms/footer.less [hunk 7]
```diff
@@ -25,7 +27,7 @@
 
     &_share-icon-list {
         margin-top: unit( @grid_gutter-width / @base-font-size-px, em );
-        .list__horizontal();
+        .m-list__horizontal();
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/organisms/footer.less]: Can we use the class in the markup rather than have the styles output twice?

File: cfgov/unprocessed/css/organisms/footer.less [hunk 8]
```diff
@@ -41,9 +43,9 @@
     }
 
     &_nav-list {
-        .list__links();
+        .m-list__links();
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/organisms/footer.less]: Same question here and throughout this file, let's use classes when we can.

File: cfgov/unprocessed/css/organisms/form.less [hunk 9]
```diff
@@ -51,8 +51,13 @@
     margin-bottom: unit( @grid_gutter-width * 2 / @base-font-size-px, em);
 
     .respond-to-max(@bp-sm-max, {
-        .btn {
+        .a-btn {
             width: 100%;
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/organisms/form.less]: This can also use the new full width button class

File: cfgov/unprocessed/css/organisms/snippet-list.less [hunk 10]
```diff
@@ -35,11 +35,11 @@
     }
 
     &-item-title {
-        .webfont-medium();
+        .u-webfont-medium();
     }
 
     &-action-list {
-        .list__horizontal();
+        .m-list__horizontal();
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/organisms/snippet-list.less]: Again here (sorry, I know this is all existing, we can push this off to another PR if you want).

File: cfgov/unprocessed/css/pages/business.less [hunk 11]
```diff
@@ -23,5 +23,5 @@
 }
 
 .business_page-content {
-    .list__download();
+    .m-list__download();
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/pages/business.less]: Is this a cfgov-refresh mixin? It doesn't exist in Capital Framework

File: cfgov/unprocessed/css/skip-nav.less [hunk 12]
```diff
@@ -20,8 +20,13 @@
 }
 
 #skip-nav:focus {
-    .btn;
-    .btn__super;
+    .a-btn;
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/skip-nav.less]: Let's not forget about moving these into the markup to avoid the duplicate styles here.

File: cfgov/jinja2/v1/_includes/molecules/related-posts.html [hunk 13]
```diff
@@ -115,9 +115,9 @@ <h3 class="h4 u-mb5">
    ========================================================================== #}
 
 {% macro _view_more() %}
-    <a class="jump-link jump-link__underline"
+    <a class="a-link a-link__underline"
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/molecules/related-posts.html]: Fixed.

Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/molecules/related-posts.html]: https://github.com/cfpb/cfgov-refresh/pull/3092/commits/05719ec1a1d134c3097ebd7172c2d50b3fad934b

File: cfgov/jinja2/v1/_includes/organisms/email-popup.html [hunk 14]
```diff
@@ -37,11 +37,11 @@
                 </div>
                 <p>
                   <a href="/owning-a-home/privacy-act-statement/" target="_blank">
-                    <span class="jump-link_text">View Privacy Act statement</span>
+                    <span class="a-link_text">View Privacy Act statement</span>
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/organisms/email-popup.html]: Removed the span in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/f13808069fbcf7ded59ba886d5170effbef81280
Also, added a code comment of where this is used.

File: cfgov/jinja2/v1/_layouts/spanish-ask-base.html [hunk 15]
```diff
@@ -83,7 +83,7 @@
         </div><!-- .brand -->
 
         <nav role="navigation" class="main-nav js-showtoggle">
-            <button class="btn js-showtoggle-trigger s-show-on-small"><img src="{{ static('nemo/_/img/icon-menu-retina.png') }}" alt="Main Menu"></button>
+            <button class="a-btn js-showtoggle-trigger s-show-on-small"><img src="{{ static('nemo/_/img/icon-menu-retina.png') }}" alt="Main Menu"></button>
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/_layouts/spanish-ask-base.html]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/f13808069fbcf7ded59ba886d5170effbef81280

File: cfgov/jinja2/v1/_layouts/spanish-ask-base.html [hunk 16]
```diff
@@ -151,10 +151,10 @@
                 <li><a href="http://youtube.com/CFPB" hreflang="en-US"><img src="{{ static("nemo/_/img/icon-youtube-retina.png") }}" alt="Youtube" class="icon-25"></a></li>
             </ul>
             <p class="s-show-on-small"><small>Un sitio web oficial del gobierno federal de <span class="usa">los Estados Unidos <img src="{{ static('nemo/_/img/us_flag_small.png') }}" alt=""></span></small></p>
-            <p><a class="btn btn-block-on-small english-link" href="/" hreflang="en-US">English</a></p>
+            <p><a class="a-btn btn-block-on-small english-link" href="/" hreflang="en-US">English</a></p>
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/_layouts/spanish-ask-base.html]: Defined here https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/legacy/static/nemo/_/c/less/es-base.less#L664-L669 Could probably be refactored to fit in our architecture better, but it's not atomized so will stand out after this PR.

File: cfgov/jinja2/v1/activity-log/_activity-list.html [hunk 17]
```diff
@@ -23,12 +23,12 @@
                     {% if is_blog(post) %}
                         {{ category_slug.render('blog',
                                 page_url,
-                                'meta-header_left',
+                                'm-meta-header_left',
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/activity-log/_activity-list.html]: That adds a float, so the spacing changes if I remove it.

File: cfgov/jinja2/v1/ask-cfpb/answer-page.html [hunk 18]
```diff
@@ -124,17 +124,18 @@ <h2 class="a-heading">
...
                 </header>
-                <ul class="list list__unstyled list__links">
+                <ul class="m-list m-list__unstyled m-list__links">
                     {% for subcat in subcategories %}
-                        <li class="list_item">
-
-                            <a href="{{slugurl('category-' + subcat.parent.slug)}}{{subcat.slug}}" class="list_link">{{ subcat.name }}</a>
+                        <li class="m-list_item">
+                            <a href="{{slugurl('category-' + subcat.parent.slug)}}{{subcat.slug}}" class="m-list_link">
+                              {{ subcat.name }}
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/ask-cfpb/answer-page.html]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/4355bc388be89b6e7a6a2ce200c793f12b3051de

File: cfgov/jinja2/v1/ask-cfpb/answer-page.html [hunk 19]
```diff
@@ -124,17 +124,18 @@ <h2 class="a-heading">
...
+                            </a>
                         </li>
                     {% endfor %}
-                    <li class="list_item">
-                      <a class="btn category-button" href="{{slugurl('category-' + category.slug)}}">
-                        See all {{category.name | lower}} questions
-                      </a>
+                    <li class="m-list_item">
+                        <a class="a-btn category-button" href="{{slugurl('category-' + category.slug)}}">
+                            See all {{category.name | lower}} questions
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/ask-cfpb/answer-page.html]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/9e8ffc0ea2a8b02be012242d4133f63c31164211

File: cfgov/unprocessed/css/activity.less [hunk 20]
```diff
@@ -37,31 +37,31 @@
...
-    .date {
+    .a-date {
         white-space: nowrap;
     }
 
     .respond-to-min(@bp-sm-min, {
-        .list_link {
+        .m-list_link {
             &:after {
                 content: '';
```
Comment (anselmbradford) [inline cfgov/unprocessed/css/activity.less]: Hmmm I see it, it appears on the items in the sidebar on /about-us/, but not sure why it's there. I removed it and couldn't see a difference. I looked at the file history too and it looks old as it predates our reorganization of the less files, so doesn't have the history of who created it originally. Probably a candidate for removal, but probably best left to another PR.

File: cfgov/unprocessed/css/enhancements/base.less [hunk 21]
```diff
@@ -1,28 +1,592 @@
+// BEGIN enhancement for normalize-legacy-addon
+
+/*
+ * Corrects list images handled incorrectly in IE 7.
```
Comment (anselmbradford) [inline cfgov/unprocessed/css/enhancements/base.less]: So this comment comes directly from https://github.com/cfpb/normalize-legacy-addon/blob/master/normalize-legacy-addon.css#L163-L171
It's an enhancement here for the situation we had a few weeks back where the bullets were showing up in the secondary nav. It should be corrected in `normalize-legacy-addon` and then removed here. So okay to keep it for this PR.

File: cfgov/unprocessed/css/molecules/global-banner.less [hunk 22]
```diff
@@ -38,7 +38,9 @@
     }
 
     .o-expandable_target {
-        .btn__secondary();
+        // Taken from .a-btn__secondary.
+        background-color: @btn__secondary-bg;
```
Comment (anselmbradford) [inline cfgov/unprocessed/css/molecules/global-banner.less]: We would have to refactor the code in CF to not use the [nested syntax](https://github.com/cfpb/capital-framework/blob/canary/src/cf-buttons/src/atoms/buttons.less#L65). In this case since it's specifically styling for the global banner, which is only used on build anyway, and the styling added for `a-btn__secondary` is minimal, I think including the styles directly makes sense vs touching CF.

File: cfgov/jinja2/v1/_includes/atoms/hyperlink.html [hunk 23]
```diff
@@ -15,4 +15,4 @@
    value.text:    A string for the text of the link.
 
    ========================================================================== #}
-<a class="jump-link" href="{{ value.url }}">{{ value.text }}</a>
\ No newline at end of file
+<a class="a-link__jump" href="{{ value.url }}">{{ value.text }}</a>
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/atoms/hyperlink.html]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/2cfd1760b2e7df96cce4ec492d0ced7a2847f39e

File: cfgov/jinja2/v1/_includes/macros/activity-snippets.html [hunk 24]
```diff
@@ -78,14 +78,17 @@
       {% set icon = category_icon.render(activity_type) %}
       <div class="activity">
           <h3 class="h4">{{ icon }} {{ header }}</h3>
-          <ul class="list list__unstyled list__spaced list__links">
+          <ul class="m-list
+                     m-list__unstyled
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/macros/activity-snippets.html]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/113e79bec50e722f9e508acb0004c64b4cf4a511

File: cfgov/jinja2/v1/_includes/molecules/global-eyebrow.html [hunk 25]
```diff
@@ -23,45 +23,46 @@
                 <span class="m-global-eyebrow_tagline-usa">United States Government</span>
             </div>
             <div class="m-global-eyebrow_actions">
-                <ul class="list__unstyled
-                           list__horizontal
+                <ul class="m-list
+                           m-list__unstyled
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/molecules/global-eyebrow.html]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/cc9f53bf3265003a4ce4ce82a604fccfebc0ede9

File: cfgov/unprocessed/css/enhancements/forms.less (hunk 3 above)
Comment (anselmbradford) [inline cfgov/unprocessed/css/enhancements/forms.less]: Removed in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/58f9e5ba585e8bf927c0a971a4b6252335aaf0b3

File: cfgov/unprocessed/css/molecules/multi-select.less (hunk 4 above)
Comment (anselmbradford) [inline cfgov/unprocessed/css/molecules/multi-select.less]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/3f3c9ab1a5e29762a3a6e40a717b86629b192c39

File: cfgov/unprocessed/css/pages/business.less (hunk 11 above)
Comment (anselmbradford) [inline cfgov/unprocessed/css/pages/business.less]: Yeah, it's a typography enhancement https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/unprocessed/css/enhancements/typography.less#L74

File: cfgov/unprocessed/css/skip-nav.less (hunk 12 above)
Comment (anselmbradford) [inline cfgov/unprocessed/css/skip-nav.less]: This one can't be since it's only added on `focus`, so it can't be there all the time.

File: cfgov/unprocessed/css/molecules/social-media.less (hunk 5 above)
Comment (anselmbradford) [inline cfgov/unprocessed/css/molecules/social-media.less]: It does use `m-list__horizontal' in the markup. This is used to inline-block the whole list, not just the list items, so that this works on one line:
screen shot 2017-07-05 at 5 21 30 pm

File: cfgov/unprocessed/css/organisms/footer.less (hunk 8 above)
Comment (anselmbradford) [inline cfgov/unprocessed/css/organisms/footer.less]: So this changes from `m-list__links()` to `m-list__horizontal` based on the screen size, so that's why it's in here as a mixin.

File: cfgov/unprocessed/css/organisms/footer.less (hunk 7 above)
Comment (anselmbradford) [inline cfgov/unprocessed/css/organisms/footer.less]: Hmmm `o-footer_share-icon-list` appears to be unused. I'll put in a code comment and it can be investigated and deleted in another PR.

File: cfgov/unprocessed/css/organisms/snippet-list.less (hunk 10 above)
Comment (anselmbradford) [inline cfgov/unprocessed/css/organisms/snippet-list.less]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/f018ff24573f53a3759a187b393fd7dc3e56183c

File: cfgov/jinja2/v1/_layouts/spanish-ask-base.html (hunk 16 above)
Comment (jimmynotjim) [inline cfgov/jinja2/v1/_layouts/spanish-ask-base.html]: Thanks. Mind leaving a note to address it so we don't forget?

File: cfgov/unprocessed/css/organisms/form.less (hunk 9 above)
Comment (anselmbradford) [inline cfgov/unprocessed/css/organisms/form.less]: So these aren't totally equivalent. `a-btn__full-on-xs` turns into a full width at a smaller size than this one does. This is kinda a weird one because it's used in the submit-button atom, which is *only* used on the conference registration form that appears as a component in wagtail. Could probably use `a-btn__full-on-xs`, but it does change the appearance so probably better to do it in another PR.

Comment (anselmbradford) [inline cfgov/unprocessed/css/organisms/form.less]: Hmm looking at the code here they look like they should be the same, but they don't behave the same when I switch them out.

File: cfgov/jinja2/v1/activity-log/_activity-list.html (hunk 17 above)
Comment (jimmynotjim) [inline cfgov/jinja2/v1/activity-log/_activity-list.html]: Hmmm, that seems really hacky, you'd expect all the other stuff to be applied as well if you're scanning the code. Mind leaving a note to address this later and either add the `m-meta-header` class or create a new class for the float rule?

File: cfgov/unprocessed/css/activity.less (hunk 20 above)
Comment (jimmynotjim) [inline cfgov/unprocessed/css/activity.less]: Cool, mind leaving a note?

File: cfgov/unprocessed/css/enhancements/base.less (hunk 21 above)
Comment (jimmynotjim) [inline cfgov/unprocessed/css/enhancements/base.less]: Sounds good.

File: cfgov/unprocessed/css/molecules/social-media.less (hunk 5 above)
Comment (jimmynotjim) [inline cfgov/unprocessed/css/molecules/social-media.less]: Oh right, thanks for the clarification.

File: cfgov/unprocessed/css/organisms/footer.less (hunk 8 above)
Comment (jimmynotjim) [inline cfgov/unprocessed/css/organisms/footer.less]: Got it, sorry for the confusion.

File: cfgov/unprocessed/css/organisms/email-popup.less (hunk 6 above)
Comment (anselmbradford) [inline cfgov/unprocessed/css/organisms/email-popup.less]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/e8b89a9d7072510b74c7fe980cebe22c7a892cca

File: cfgov/unprocessed/css/enhancements/forms.less [hunk 26]
```diff
@@ -25,7 +25,7 @@ select[multiple] {
     - cf-forms
 */
 
-@input-warning:     @gold;
+/*@input-warning:     @gold;
```
Comment (anselmbradford) [inline cfgov/unprocessed/css/enhancements/forms.less]: Shoot I can't recall why I commented this out! But I can't find a use for it. Removed in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/4c234e9094b3ba140bc8e5df91c154f457072e34

File: cfgov/unprocessed/css/cf-theme-overrides.less [hunk 27]
```diff
@@ -222,35 +222,37 @@
 
 // Color variables
```
Comment (anselmbradford) [inline cfgov/unprocessed/css/cf-theme-overrides.less]: 👍  own PR

File: cfgov/jinja2/v1/index.html [hunk 28]
```diff
@@ -174,15 +174,15 @@ <h3 class="h1 m-card_heading">97% of consumers</h3>
                         <section class="content-l_col
                                         content-l_col-1-3">
                             {% for block in sections.value %}
-                                <div class="m-related-posts_list-container">
+                                <div class="m-related-posts_list-container m-list">
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/index.html]: So the `m-list_item` is used to make the links bold, so it is used, but you bring up a good point that perhaps it should not be. I can't change it here though without refactoring the component.

File: cfgov/unprocessed/css/organisms/form.less (hunk 9 above)
Comment (jimmynotjim) [inline cfgov/unprocessed/css/organisms/form.less]: Odd, leave it as is

File: cfgov/jinja2/v1/_includes/organisms/footer.html [hunk 29]
```diff
@@ -61,61 +61,61 @@
         <div class="o-footer-middle-left">
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/organisms/footer.html]: Let's do this one in another PR :) Added a note for it.

File: cfgov/jinja2/v1/_layouts/spanish-ask-base.html (hunk 16 above)
Comment (anselmbradford) [inline cfgov/jinja2/v1/_layouts/spanish-ask-base.html]: Added!

**Comment by anselmbradford** [comment]
@jimmynotjim I rebased the updates from master in so you may need to delete and re-pull, but I think it's ready for review again. I think the only ones I didn't touch were the ones that would need significant refactoring and are best suited to their own PR to minimize migration bugs.

File: cfgov/unprocessed/css/activity.less (hunk 20 above)
Comment (anselmbradford) [inline cfgov/unprocessed/css/activity.less]: Added!

File: cfgov/unprocessed/css/molecules/global-banner.less (hunk 22 above)
Comment (jimmynotjim) [inline cfgov/unprocessed/css/molecules/global-banner.less]: Got it

File: cfgov/jinja2/v1/index.html (hunk 28 above)
Comment (jimmynotjim) [inline cfgov/jinja2/v1/index.html]: That's fine, let's leave a note here too (sorry to ask you to leave so many, I just worry we're going to lose track in the future).

File: cfgov/jinja2/v1/activity-log/_activity-list.html (hunk 17 above)
Comment (anselmbradford) [inline cfgov/jinja2/v1/activity-log/_activity-list.html]: Added! Also, misread your original comment and I tried adding a `m-meta-header` and that changes the layout too.

File: cfgov/jinja2/v1/_layouts/spanish-ask-base.html (hunk 16 above)
Comment (anselmbradford) [inline cfgov/jinja2/v1/_layouts/spanish-ask-base.html]: Added in es-base.less!

File: cfgov/unprocessed/css/molecules/notification.less (hunk 1 above)
Comment (jimmynotjim) [inline cfgov/unprocessed/css/molecules/notification.less]: Should cf-icon be doing that?

File: package.json (hunk 2 above)
Comment (anselmbradford) [inline package.json]: Updated in latest!

File: cfgov/jinja2/v1/index.html (hunk 28 above)
Comment (anselmbradford) [inline cfgov/jinja2/v1/index.html]: Added! No problem, I love TODOs for when between things.

**Comment by Scotchester** [comment]
I'm starting my final look at this, but one of the acceptance tests is timing out in Travis.

**Comment by anselmbradford** [comment]
@Scotchester it's because the search box (when expanded) is borked. I'll have a fix in momentarily. Almost there, just the button isn't working.

**Comment by anselmbradford** [comment]
@Scotchester @jimmynotjim @sebworks Global search is updated. I made one slight adjustment to fix the misaligned search icon and search text, which has been bugging me since last year!
screen shot 2017-07-06 at 11 44 25 am
screen shot 2017-07-06 at 11 44 20 am

It uses the existing clear label, which will need to be reconciled with the clear button in CF later.

File: package.json [hunk 30]
```diff
@@ -22,7 +22,15 @@
...
-    "capital-framework": "3.6.1",
+    "cf-buttons": "4.4.1",
+    "cf-core": "4.3.1",
+    "cf-expandables": "4.1.3",
+    "cf-forms": "4.3.2",
+    "cf-grid": "4.2.1",
+    "cf-icons": "4.1.1",
+    "cf-layout": "4.4.1",
+    "cf-pagination": "4.1.3",
+    "cf-typography": "4.1.2",
```
Comment (anselmbradford) [inline package.json]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/c8e229829bf7428dc0075c4a05d416ad41b75c85

**Comment by sebworks** [comment]
@anselmbradford, what are doing about the on-demand header / footer ?

**Comment by sebworks** [comment]
I'll run more test but merge at will. It looks great.

File: cfgov/unprocessed/css/molecules/notification.less (hunk 1 above)
Comment (anselmbradford) [inline cfgov/unprocessed/css/molecules/notification.less]: Not sure. The icon was shifted down and all I could find when comparing vs the live site was that `line-height: 1` was set via cf-icon in the live site and wasn't locally. This was the only spot I saw an issue with it.

**Comment by anselmbradford** [comment]
> After running ./frontend.sh, when I run the server, I'm seeing u-move-left attached to the mega menu on a wide display, causing it to be mostly off the screen:

@Scotchester can you verify that you can repeat this?

**Comment by jimmynotjim** [comment]
I can't reproduce the bug @Scotchester reported

File: cfgov/jinja2/v1/_includes/molecules/info-unit.html [hunk 31]
```diff
@@ -90,16 +90,16 @@
...
+                <li class="m-list_item">
                     {% if link.text is defined and link.text.find('@') > -1 %}
                         {% if not value.emails is defined %}
                             {% do value.update({'emails':[{'url':link.text}]}) %}
                         {% endif %}
                         {% include 'contact-email.html' with context %}
                     {% else %}
-                        {% set link_class = 'btn m-info-unit_btn' if
+                        {% set link_class = 'a-btn m-info-unit_btn' if
                             value.is_button else 'list_link' %}
```
Comment (Scotchester) [inline cfgov/jinja2/v1/_includes/molecules/info-unit.html]: The class in this `else` clause should be `m-list_link`.

File: cfgov/jinja2/v1/activity-log/_activity-list.html (hunk 17 above)
Comment (Scotchester) [inline cfgov/jinja2/v1/activity-log/_activity-list.html]: Hmm... we perhaps shouldn't have added `float: left;` to `meta-header_left` [when making it atomic](https://github.com/cfpb/capital-framework/commit/6662506bbfcea95f739e9e0c1689ec4fac83b908#diff-fbe01bf95b92595df0d955810b71204dR11)

Having that be in there results in this unsightliness:

screen shot 2017-07-11 at 11 40 33

It does nothing of value right now, so as a temporary measure, I'd recommend removing the `m-meta-header_left` classes from the resulting markup.

**Comment by anselmbradford** [comment]
> It does nothing of value right now, so as a temporary measure, I'd recommend removing the m-meta-header_left classes from the resulting markup.

Removed in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/9157ce82bc35ead077c009c162dad5bc09f2e093

File: cfgov/jinja2/v1/_includes/molecules/info-unit.html (hunk 31 above)
Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/molecules/info-unit.html]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/8e4ecd023291894fce0e90444f12f5d7b0319045

**Comment by Scotchester** [comment]
Oh, and for the record, my issue with the oddly-placed menu went away.
